lib/rubocop/cop/style/frozen_string_literal_comment.rb

```diff
@@ -3,11 +3,12 @@
 module RuboCop
   module Cop
     module Style
-      # This cop is designed to help upgrade to after Ruby 3.0. It will add the
-      # comment `# frozen_string_literal: true` to the top of files to
-      # enable frozen string literals. Frozen string literals may be default
-      # after Ruby 3.0. The comment will be added below a shebang and encoding
-      # comment. The frozen string literal comment is only valid in Ruby 2.3+.
+      # This cop is designed to help mutable string literals to frozen string
```

Comment:
to help you transition from mutable...

I guess we can still add some explanation that there's a chance that string literals in Ruby will become frozen by default at some point in the future.

Reply:
Thanks for your review. I updated the description.